modules/plugins/assistant/codecompanion/config.nix

```
@@ -3,9 +3,21 @@
   lib,
   ...
```

```suggestion
  options,
  ...
```

then you can access the mappings option as `options.vim.assistant.codecompanion-nvim.mappings`

see run-nvim for a full example